src/index.js

```diff
@@ -72,15 +80,51 @@ export function filterCards(cards, options) {
 	return cards.filter(card => !(options.ignoreLevels || []).includes(card.headingLevel));
 }
 
-export function deckFromCards(cards, options) {
+export function deckFromCards(cards, images, options) {
 	// create new deck
 	const apkg = new AnkiDeck(options.deckName, { css: '#front * {margin:0; padding:0;}' }); // ToDo: move CSS to different file
 	console.log(`deck initialized!`);
+	// add media files to deck
+	images.forEach(image => {
+		apkg.addMedia(image.filteredPath, fs.readFileSync(image.filePath));
+	});
 	// add cards to deck (convert tokens to html)
 	cards.forEach((card, i) => {
 		const { front, back } = card.renderToHTML(md);
 		apkg.addCard(front, back);
 	});
 	console.log(`added ${cards.length} cards to the deck!`);
+	console.log(`added ${images.length} images to the deck!`);
 	return apkg;
 }
+
+export function imagesFromTokens(tokens, inputPath) {
+	let images = [];
+	tokens.forEach((token) => {
+		if (token.type === 'image') {
+			// find images in tokens and save them to a list
+			let attrPath = token.attrGet('src');
+			// make the image path relative
```

Comment:
```suggestion
			if (attrPath /* is-url */) continue;
			// make the image path relative
```

Ich würde hier ja gern ein bisschen Regex-Magic hinschreiben, aber leider habe ich mich damit noch nie intensiv auseinandergesetzt. Steht aber in meinem Backlog😅

Reply:
Das war aber nen schneller Fix 😂

Reply:
Ah jo, das habe vor 30 sekunden  eingefügt 😁
Allerdings habe ich nicht so viel Regex betrieben und einfach nach `://` geschaut

Reply:
Meine Zeile finde ich aber schöner als ein weiteres nested-if, muss ich sagen. Oder macht das die Rekursion mit `inline`-Elementen kaput?

Reply:
Hatte es mit `continue` probiert aber das wollte nicht so recht.

`Uncaught SyntaxError: Illegal continue statement: no surrounding iteration statement` bekomme ich da.

Reply:
Eigentlich sollte da nichts passieren, dass `inline` könnte doch auch in ein `else if` 🤔

Reply:
Ich habs!

Man muss `return` schreiben, weil jede Schleifeniteration eine abgeschlossene Funktion ist.